src/containerapp/azext_containerapp/_clients.py

```diff
@@ -887,7 +900,7 @@ def list_auth_token(cls, cmd, builder_name, build_name, resource_group_name, loc
 
 
 class JavaComponentPreviewClient():
-    api_version = AUG_PREVIEW_API_VERSION
+    api_version = PREVIEW_API_VERSION
```

Comment:
this line can be removed, right? can we use `cls.api_version` like other clients?

Reply:
This line if for define the api-version this client using, cannot be removed.